programs/drift/src/instructions/keeper.rs

```diff
@@ -627,6 +635,7 @@ pub fn handle_place_signed_msg_taker_order<'c: 'info, 'info>(
     )?;
 
     let high_leverage_mode_config = get_high_leverage_mode_config(&mut remaining_accounts)?;
+    let escrow = get_revenue_share_escrow_account(&mut remaining_accounts)?;
```

Comment:
would be good to make sure that we do the maker map, revenue share and hlm config in same order for every tx

since this is replacing referrer which was previously in maker map, maybe we do:
- maker map
- revenue share
- hlm

wdyt?